lab11-amanda/test/note-router-test.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+require('dotenv').config({path: `${__dirname}/../.env`});
+const superagent = require('superagent');
+const expect = require('expect');
+const server = require('../lib/server.js');
+const API_URL = `http://localhost:${process.env.PORT}`;
+let tempNote;
+
+describe('testing note routes', () => {
+  before(server.start);
+  after(server.stop);
+
+  describe('test POST /api/notes',() => {
+    it('should respond with a note', () => {
+      superagent.post(`${API_URL}/api/notes`)
```

Comment:
Fixed it:
      return superagent.post(`${API_URL}/api/notes`)
      .send(data)